README.md

```diff
@@ -132,13 +140,6 @@ can add the following line to `~/.profile`:
 . /opt/intel/sgxsdk/environment
```

Comment:
As far as I can recall form my experience, sourcing `$SGX_SDK/environment` in `profile` was not enough. Modern Ubuntu system seems to reset `LD_LIBRARY_PATH` after processing commands from `profile`. That time I found `ld.so.conf` was the preferred way of telling the system to search for dynamic libraries in an additional directory. Could you please check that? If it is so, then let's just mention here that `sudo ldconfig` should be invoked once after adding the file.

Reply:
> Modern Ubuntu system seems to reset LD_LIBRARY_PATH after processing commands from profile. That time I found ld.so.conf was the preferred way of telling the system to search for dynamic libraries in an additional directory.

You're right. I found [the related notice](https://help.ubuntu.com/community/EnvironmentVariables#File-location_related_variables), that says

> LD_LIBRARY_PATH: ... Since Ubuntu 9.04 Jaunty Jackalope, LD_LIBRARY_PATH cannot be set in $HOME/.profile, /etc/profile, nor /etc/environment files. You must use /etc/ld.so.conf.d/*.conf configuration files.

And ldconfig seems to be called within installation of PSW .deb package, so I'll add some to make it clear that `sudo ldconfig` is required for simulation mode.

Thanks!